src/script_chainer/config/script_config.py

```
@@ -141,6 +168,64 @@ def __init__(self, module_name: str, is_mock: bool = False):
+    def save_python_script(self, idx: int, code: str) -> str:
+        path = self.script_list[idx].script_path
+        if not path:
+            path = self.get_python_script_path(idx)
+            self.script_list[idx].script_path = path
+            self.save()
+        with open(path, 'w', encoding='utf-8') as f:
+            f.write(code)
+        return path
+
+    def _next_python_script_number(self) -> int:
+        """获取下一个可用的 Python 脚本编号（从已有文件名推算）。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)